Bteq web initial version running on the react.js and node.js

Arman199116 opened the pull request.

Arman199116 committed:
Bteq web initial version running on the react.js and node.js

maryantonyan reviewed (changes requested):
The output div should be cleared, not overwritten.

**_Steps to reproduce:_**
```
.set width 50;
select 1, 2, 3, 4, 5;
->Submit
```
```
.set width a;
select 1, 2, 3, 4, 5;
->Submit
```

![image](https://user-images.githubusercontent.com/15236153/69244577-9df51d80-0bbe-11ea-976e-653c9a689760.png)

Arman199116 committed:
changed 'localhost' to '127.0.0.1' in 'read/package.json' file
added 'componentWillUnmount' function, becouse need to delete 'setInterval' timerId
and 'res.end()', when 'out.bteq' file not exists, as a to 'get' : response

Arman199116 committed:
clear output 'pre', until you get a new value

Arman199116 committed:
update css grid style
